src/transformers/models/gpt_bigcode/configuration_gpt_bigcode.py

```diff
@@ -0,0 +1,182 @@
+# coding=utf-8
+# Copyright 2023 The BigCode team and HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" GPTBigCode configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+GPT_BIGCODE_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "bigcode/santacoder-fast-inference": "https://huggingface.co/bigcode/santacoder-fast-inference/resolve/main/config.json",
```

Comment:
Can we add the tokenizer too? Not sure how to do it since we still use the GPT2 tokenizer